package.json

```diff
@@ -83,7 +83,7 @@
     "build": "yarn gatsby build",
     "clean": "yarn gatsby clean",
     "develop": "yarn gatsby develop",
-    "develop:debug": "GATSBY_DEBUG_MEDIA_QUERIES=true yarn develop",
+    "develop:debug": "yarn run test && GATSBY_DEBUG_MEDIA_QUERIES=true yarn develop",
```

Comment:
I don't think we should do this. It's adding a command that is hidden. Let's keep our commands as simple and declarative as possible.

I'm also thinking we should have a simple `GATSBY_ENV` variable that simply puts us automatically in a particular state within the project, e.g. `debug` vs `development` vs `production`.